Costumer.py

```diff
@@ -0,0 +1,17 @@
+class Costumer(object):

```

Comment:
doc 

see here for details https://github.com/osherdp/python-training/blob/master/README.md